src/web/CustomScrollbar.ts

```diff
@@ -329,7 +329,24 @@ export class Scrollbar {
     }
 
     private _handleMouseDown(e: React.MouseEvent<any>) {
-        this._prevent(e);
+        const target = e.currentTarget as any;
+
+        if (this._dragging === true || !target) {
```

Comment:
It's redundant to compare a boolean value to the value "true", and it makes the code less readable.